stdlib/public/core/Collection.swift

```diff
@@ -234,17 +234,19 @@ extension CollectionType where SubSequence == Self {
     self = self[startIndex.successor()..<endIndex]
     return element
   }
+}
 
+extension CollectionType where
+    SubSequence == Self, Index: BidirectionalIndexType {
```

Comment:
Will do. Is there a doc for stdlib coding conventions (indentation, etc.)? I couldn't find one.